dev/Common/NotificationTelemetryHelper.h

```diff
@@ -6,23 +6,25 @@
 class NotificationTelemetryHelper
 {
 public:
+    NotificationTelemetryHelper()
+    {
+        m_isPackagedApp = AppModel::Identity::IsPackagedProcess();
+        m_appName = m_isPackagedApp ? GetAppNamePackaged() : GetAppNameUnpackaged();
+    }
+
     inline bool ShouldLogEvent()
     {
         return c_maxEventLimit >= UpdateLogEventCount();
     }
 
-    inline bool IsPackagedApp() const
+    inline const bool IsPackagedApp() const
```

Comment:
Why return a const bool and not just a bool> It's not like your returning a ref or a pointer here.

Reply:
Thank you for the advice! I realized that setting const to a thing returned by value isn't really needed.